worker/stripe_charge_succeeded.js

```diff
@@ -11,10 +30,14 @@ module.exports = function(notifier_messager, mailroom) {
     }
 
     var currency_code = event.data.currency.toUpperCase();
+    // If this is a zero decimal currency then use it directly
+    // Otherwise divide by 100 to get currency fractional value
+    var amount = zeroDecimalCurrencies.indexOf(currency_code) !== -1 ?
+      amount : (event.data.amount / 100).toFixed(2);
```

Comment:
Do we need the toFixed or are we just being safe here?

I *think* / 100 on a whole number will always produce two or less decimal places?